adcm_client/objects.py

```diff
@@ -648,9 +647,14 @@ class Action(BaseAPIObject):
     config = None
 
     def _get_config(self):
-        config = {}
+        config = dict()
```

Comment:
Why was {} replaced by dict()

Reply:
I rewrote this peace several times and just of my personal preferences wrote this way.

Reply:
Ok.

Reply:
Btw. Literal using is faster, but there is a difference only when we compare between several million iterations. If it is one like in our case i don`t think that there are any actual reasons to chose literal using or function call.

Reply:
You have to write 4 characters more.